LICENSE.txt

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
-
-Copyright (c) 2016 Martin Leech
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
```

Comment:
Per the MIT license, we need to maintain the original license even if it's a derivite work.

Reply:
Resolved in [e24f2e1](https://github.com/EasyPost/scotch/pull/1/commits/e24f2e11e88126ae38a5543c9b7226db10764b7e)